.npmignore

```diff
@@ -0,0 +1,2 @@
+/.eslintrc.cjs
```

Comment:
this is needed because eslint-plugin-n warns us when we require from a dependency in a file that we ship to npm and that dependency is not declared in "dependencies".